gulpfile.babel.js

```diff
@@ -0,0 +1,33 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const webpack = require('webpack');
+
+const webpackConfig = require('./webpack.config.js');
+
+webpackConfig.plugins.push(new webpack.optimize.DedupePlugin());
+webpackConfig.plugins.push(new webpack.optimize.UglifyJsPlugin({
+  compress: {
+    warnings: false
+  }
+}));
```

Comment:
Not sure how great this implementation is. Maybe I should put into some `productionPlugins` array and then do something like:
```js
const productionPlugins = [
  new webpack.optimize.DedupePlugin(),
  new webpack.optimize.UglifyJsPlugin({
    compress: {
      warnings: false
    }
  })
];

webpackConfig.plugins = _.union(webpackConfig.plugins || [], productionPlugins);
```